src/main/java/io/odpf/firehose/metrics/Metrics.java

```diff
@@ -37,6 +38,19 @@
     public static final String SINK_HTTP_RESPONSE_CODE_TOTAL = APPLICATION_PREFIX + SINK_PREFIX + HTTP_SINK_PREFIX + "response_code_total";
     public static final String SINK_PUSH_BATCH_SIZE_TOTAL = APPLICATION_PREFIX + SINK_PREFIX + "push_batch_size_total";
 
+    // OBJECT STORAGE SINK MEASUREMENTS
+    public static final String SINK_OBJECT_STORAGE_RECORD_PROCESSED_TOTAL = APPLICATION_PREFIX + SINK_PREFIX + OBJECTSTORAGE_SINK_PREFIX + "record_processed_total";
+    public static final String SINK_OBJECT_STORAGE_RECORD_PROCESSING_FAILED_TOTAL = APPLICATION_PREFIX + SINK_PREFIX + OBJECTSTORAGE_SINK_PREFIX + "record_processing_failed_total";
+
+    public static final String SINK_OBJECT_STORAGE_LOCAL_FILE_OPEN_TOTAL = APPLICATION_PREFIX + SINK_PREFIX + OBJECTSTORAGE_SINK_PREFIX + "file_open_total";
```

Comment:
for files_open, guage should be the right metric, so that we know how many active files are open at a given point in time.

Reply:
will remove, `SINK_OBJECT_STORAGE_RECORD_PROCESSED_TOTAL` ,  `SINK_OBJECT_STORAGE_RECORD_PROCESSING_FAILED_TOTAL` , and scope tags

Reply:
we can have this as a gauge metric and send it on every local file checker loop.

Reply:
changed to gauge, and moved into local file checker loop